pkg/api/validation/dynakube/testdata/latest-default.yaml

```diff
@@ -262,6 +260,8 @@ spec:
       - maxSkew: 1
         topologyKey: kubernetes.io/hostname
         whenUnsatisfiable: ScheduleAnyway
+    sqlExtensionExecutor:
```

Comment:
We need to update the official document accordingly, since I've documented in the release branch using previous names. cc @StefanHauth